### ISSUE TITLE
Show symbolic link files using a different visual representation

### ISSUE THREAD

**Issue by nekohayo** [cboxdoerfer/fsearch#447]
I have a lot of soft symlinks of various files in my folders, so that they can be found from various places while browsing with Nautilus.

However, they all show up as identical search results in FSearch. There should be some visual distinction to tell them apart from the original, "real" file that the symlinks are pointing.

Nautilus simply shows an arrow emblem, as you can see in the upper half of this screenshot (in this particular case, it also shows the read-only permission indicator emblem):

![image](https://user-images.githubusercontent.com/479401/205462786-bd19fc21-723e-4d8d-b832-b237e1b03a64.png)

The lower half is how fsearch currently represents the same results.

Another potential (perhaps simpler to implement) approach would be to make the row 50% opaque (or maybe up to 70% opaque) to denote that it is a secondary result to the original.

**Comment by cboxdoerfer** [comment]
Yes, that's been bothering me as well. Symbolic links definitely should be highlighted in one way or another. The emblems look kind of nice I think and aren't much work to implement.

It would also be great to show the link target, for example while hovering over the emblem.

So yes, that's something I'm going to add.

**Comment by JohnFitz** [comment]
@nekohayo Definitely agree with this. As a temporary workaround, you could make yourself a filter.
I have a filter called "No Links" with this query `!contenttype:inode/symlink` with a macro name of `nl`

The only caveat is that it causes FSearch to churn a lot, so it's better if you search for the doc first to narrow down the result set, then add the macro i.e. `better invoice nl:` rather than `nl: better invoice`
